apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -737,28 +740,43 @@ where
                             self.storage.last_height,
                         ) {
                             response.log = String::from(VALID_MSG);
+                            // validator set update votes should be decided
+                            // as soon as possible
+                            response.priority = i64::MAX;
                         } else {
                             response.code = 1;
-                            response.log = String::from(
-                                "Invalid validator set update vote extension",
+                            response.log = format!(
+                                "{INVALID_MSG}: Invalid validator set update \
+                                 vote extension",
                             );
                         }
                     }
                     Ok(TxType::Protocol(ProtocolTx { tx, .. })) => {
                         response.code = 1;
                         response.log = format!(
-                            "The following protocol tx cannot be added to the \
-                             mempool: {tx:?}"
+                            "{INVALID_MSG}: The following protocol tx cannot \
+                             be added to the mempool: {tx:?}"
```

Comment:
nit: it was already like this before this PR, but I'm unsure if we should debug stringify the `tx` and put it in the response message because it could be huge, "{INVALID_MSG}: Forbidden protocol transaction" or something similar might be ok